src/lib/components/ZTicketDisplay.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+	import IconCheck from '~icons/ant-design/check-outlined';
+	import IconArrowLeft from '~icons/ant-design/arrow-left-outlined';
+
+	export let zTicketText: string;
+	export let sessionInfo: {
+		openedAt: Date;
+		closedAt?: Date | null;
+	};
+	export let showBackToPos = false;
+
+	let copied = false;
+
+	function printTicket() {
+		window.print();
+	}
+
+	function copyToClipboard() {
+		const text = document.querySelector('.printable')?.textContent || '';
```

Comment:
Is it really expected to use an empty string if we can't find the content to copy?

I think this should be:
```
const element = document.querySelector('.printable');
if (element) {
  ...
} else {
  console.error(...);
}
```